blockchain/blockchain.go

```diff
@@ -24,10 +24,8 @@ import (
 	"github.com/onflow/cadence/runtime"
 	"github.com/onflow/cadence/runtime/common"
 	"github.com/onflow/cadence/runtime/interpreter"
-	"github.com/onflow/flow-go-sdk"
-	sdk "github.com/onflow/flow-go-sdk"
+	flowgosdk "github.com/onflow/flow-go-sdk"
```

Comment:
```suggestion
	flowsdk "github.com/onflow/flow-go-sdk"
```
The Go part can be dropped, since we are in Go language.